### PR TITLE
move data files to priv so they're accessible in releases

### PR DESCRIPTION
**Pull request by stocks29**
This PR fixes https://github.com/SebastianSzturo/countries/issues/31 by moving all the data files to `priv/data` and updating the code to load data from there so it's easily accessible regardless of how the app is packaged (ie `mix release` works).

### CODE REVIEW

**Comment by stocks29** [comment]
@SebastianSzturo , thoughts on merging this? :)

File: lib/loader.ex [hunk 1]
```diff
@@ -4,25 +4,23 @@ defmodule Countries.Loader do
...
     |> Enum.flat_map(fn code ->
-         :yamerl.decode_file data_path("countries/#{code}.yaml")
+         :yamerl.decode_file data_path(["countries", "#{code}.yaml"])
        end)
     |> Enum.map(&convert_country/1)
   end
 
   defp data_path(path) do
-    Path.join(@data_path, path)
+    Path.join([:code.priv_dir(:countries), "data"] ++ path)
```
Comment (steffenix) [inline lib/loader.ex]: Not working for me on ci.
Will try to use:
`Path.join(Mix.Project.deps_paths[:countries] || File.cwd!, "priv")`

**Comment by SebastianSzturo** [comment]
@stocks29 Sorry, I don't have much time maintaining this library. Would you be interested in taking a maintainer role on the repo to take care of it?

**Comment by steffenix** [comment]
I would like to help.

**Comment by stocks29** [comment]
I don't have a ton of time, but I'd be happy to help out here and there as I can.

**Comment by SebastianSzturo** [comment]
Thanks a lot! I feel bad for this library to be stalled. I added you both as collaborators. Feel free to merge anything when you reviewed each-others PRs. 

Feel free to open an issue and ping me when you are happy with the updates and need me to push a new version to hexpm. 👍

**Pull request merged**